README.md

```diff
@@ -613,13 +613,34 @@ type Account struct {
 ### Description of struct
 
 ```go
+// Account model info
+// @Description User account information
+// @Description with user id and username
 type Account struct {
```

Comment:
I believe you should document the contextual is ignored 
```go
// This comment line will be ignored
```

Reply:
Do you say about `// Account model info`?

Reply:
Yes.